helm/reana/templates/reana-message-broker.yaml

```diff
@@ -5,15 +5,24 @@ metadata:
   name: {{ include "reana.prefix" . }}-message-broker
   namespace: {{ .Release.Namespace }}
 spec:
+  type: "NodePort"
   ports:
    - port: 5672
      targetPort: 5672
      name: "tcp"
      protocol: TCP
+  {{- if .Values.debug.enabled }}
+   - port: 32585
+     targetPort: 15672
+     nodePort: 32585
+     name: "management"
+     protocol: TCP
+  {{- else }}
    - port: 15672
```

Comment:
I see that we are already exposing `15672` port, why do we need to expose also `32585` ?

Reply:
You are right that we are exposing `15672` but that's only for internal communication but now we want to expose externally, that's why we need to forward it to [`nodePort`](https://kubernetes.io/docs/concepts/services-networking/service/#nodeport) and according to docs there is range of ports that we can define so I just randomly choose `32585`.

Reply:
The convention was to use 30000 as a base and append the usual port, hence 30080/http and 30443/https for web UI (corresponding to ports 80 and 443 by default), 31984 for wdb (corresponding to port 1984 by default)... Using this technique would call for using 35672 (or even 45672, but I guess it's better to stay in the 3xxxxx range) for RabbitMQ.

Reply:
Retesting after changes, I get:

```
Error: Service "reana-message-broker" is invalid: spec.ports[1].nodePort: 
Invalid value: 35672: provided port is not in the valid range. 
The range of valid ports is 30000-32767
```

We would have to amend Kubertenes's API server settings to allow higher ports, which I think is not worth the trouble.... So what about changing to 31672?